### PR TITLE
allowing true= and false= command parameter tags

### PR DESCRIPTION
**Pull request by scottfrazer**
https://broadinstitute.atlassian.net/browse/DSDEEPB-1818

### CODE REVIEW

**Comment by mcovarr** [comment]
:+1:

File: src/main/scala/cromwell/binding/command/ParameterCommandPart.scala [hunk 1]
```diff
@@ -38,6 +38,7 @@ case class ParameterCommandPart(attributes: Map[String, String], expression: Wdl
     }
 
     value match {
+      case b: WdlBoolean if attributes.contains("true") && attributes.contains("false") => if (b.value) attributes.get("true").head else attributes.get("false").head
```
Comment (kshakir) [inline src/main/scala/cromwell/binding/command/ParameterCommandPart.scala]: Can you add a negative expectation test showing what happens if one forgets true or false?

**Comment by kshakir** [comment]
One additional test request, then feel free to merge. :+1:

File: src/main/scala/cromwell/binding/command/ParameterCommandPart.scala (hunk 1 above)
Comment (scottfrazer) [inline src/main/scala/cromwell/binding/command/ParameterCommandPart.scala]: Will do

**Pull request merged**